features/admin_clicks_cat_to_see_details.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Link categories to categories show for an Admin
```

Comment:
Now that this feature covers more than just _admins_, I think we should adjust the feature description.

[This is a short description of what this whole file covers:  Think of it as how to describe what this file covers to someone that asks; how you would answer "what is the purpose of this file?"]

My suggestion: `Link categories to the categories show page for Admins only'  
(or `... only for Admins`)

Other suggestions or ideas for this?